### PR TITLE
`solana-ledger-tool verify` now runs AccountsBackgroundService

### PR DESCRIPTION
**Pull request by mvines**
#### Problem
Jeff's machine is OOMing

#### Summary of Changes
Try to do Jeff a solid

### CODE REVIEW

File: ledger/src/bank_forks_utils.rs [hunk 1]
```diff
@@ -61,16 +67,57 @@ pub fn load(
...
+    {
+        let root_bank = bank_forks.read().unwrap().root_bank();
+        root_bank.set_callback(Some(Box::new(
+            root_bank
+                .rc
+                .accounts
+                .accounts_db
+                .create_drop_bank_callback(pruned_banks_sender),
+        )));
+    }
```
[the remark this answers is not in the document]
Comment (mvines) [reply ledger/src/bank_forks_utils.rs:89]: Yeah that makes sense. I'd like to get feedback that this approach helps reduce the RAM usage he was seeing first

[the remark this answers is not in the document]
Comment (steviez) [reply ledger/src/bank_forks_utils.rs:89]: I ran into same issue of OOM'ing when running `solana-ledger-tool verify` with tip of master (e74ad90cdb574c8b8598418a4591804e8cf61b33). Sadly, cherry-picking this PR on top didn't help

[the remark this answers is not in the document]
Comment (steviez) [reply ledger/src/bank_forks_utils.rs:89]: In light of https://github.com/solana-labs/solana/issues/26895#issuecomment-1203288827, I'll try this PR on top of slightly newer tip of master (https://github.com/solana-labs/solana/commit/224550d65fca059a46f6293d41cee7055986a638) a shot

File: ledger/src/bank_forks_utils.rs [hunk 2]
```diff
@@ -61,16 +67,57 @@ pub fn load(
...
+        snapshot_request_handler: None,
+        pruned_banks_receiver,
+    };
+    let exit = Arc::new(AtomicBool::new(false));
+
+    let accounts_background_service = AccountsBackgroundService::new(
+        bank_forks.clone(),
+        &exit,
+        abs_request_handler,
+        false,
```
Comment (steviez) [inline ledger/src/bank_forks_utils.rs:101]: This line is what prevented this from working; this `false` corresponds to `accounts_db_caching_enabled` arg, so cleanup wasn't happening as expected.

**Review by steviez — COMMENTED** [review]
I took over this PR in https://github.com/solana-labs/solana/pull/26914

**Pull request closed without merging**